apps/client/src/views/backstage/Backstage.tsx

```diff
@@ -0,0 +1,178 @@
+    time,
+    isMirrored,
+    runtime,
+    selectedId,
+    settings,
+  } = props;
+
+  const { getLocalizedString } = useTranslation();
+  const { secondarySource } = useBackstageOptions();
+  const [blinkClass, setBlinkClass] = useState(false);
+  const { height: screenHeight } = useViewportSize();
+
+  useWindowTitle('Backstage');
+
+  // blink on change
+  useEffect(() => {
+    setBlinkClass(false);
+
+    const timer = setTimeout(() => {
+      setBlinkClass(true);
+    }, 10);
+
+    return () => clearTimeout(timer);
+  }, [selectedId]);
+
+  // gather card data
+  const hasEvents = backstageEvents.length > 0;
+  const { showNow, nowMain, nowSecondary, showNext, nextMain, nextSecondary } = getCardData(
+    eventNow,
+    eventNext,
+    'title',
+    secondarySource,
+    time.playback,
+  );
+
+  // gather timer data
+  const clock = formatTime(time.clock);
+  const isPendingStart = getIsPendingStart(time.playback, time.phase);
+  const startedAt = isPendingStart ? formatTime(time.secondaryTimer) : formatTime(time.startedAt);
+  const scheduledStart =
+    hasEvents && showNow ? '' : formatTime(runtime.plannedStart, { format12: 'hh:mm a', format24: 'HH:mm' });
+  const scheduledEnd =
+    hasEvents && showNow ? '' : formatTime(runtime.plannedEnd, { format12: 'hh:mm a', format24: 'HH:mm' });
+
+  let stageTimer = millisToString(time.current, { fallback: timerPlaceholderMin });
+  stageTimer = removeLeadingZero(stageTimer);
+
+  // gather presentation styles
+  const qrSize = Math.max(window.innerWidth / 15, 72);
+  const showProgress = getShowProgressBar(time.playback);
+  const showSchedule = hasEvents && screenHeight > 700; // in vertical screens we may not have space
+
+  // gather option data
+  const defaultFormat = getDefaultFormat(settings?.timeFormat);
+  const backstageOptions = getBackstageOptions(defaultFormat, customFields);
+
+  return (
+    <div className={`backstage ${isMirrored ? 'mirror' : ''}`} data-testid='backstage-view'>
+      <ViewParamsEditor viewOptions={backstageOptions} />
+      <div className='project-header'>
+        {general?.projectLogo ? <ViewLogo name={general.projectLogo} className='logo' /> : <div className='logo' />}
+        <div className='title'>{general.title}</div>
+        <div className='clock-container'>
+          <div className='label'>{getLocalizedString('common.time_now')}</div>
+          <SuperscriptTime time={clock} className='time' />
+        </div>
+      </div>
+
+      {showProgress && <ProgressBar className='progress-container' current={time.current} duration={time.duration} />}
+
+      {!hasEvents && <Empty text={getLocalizedString('common.no_data')} className='empty-container' />}
+
+      <div className='card-container'>
+        {showNow && (
+          <div className={cx(['event', 'now', blinkClass && 'blink'])}>
+            <TitleCard title={nowMain} secondary={nowSecondary} />
+            <div className='timer-group'>
+              <div className='aux-timers'>
```

Comment:
This has nothing to do with the actual aux timer right?
could it get a different name?

Reply:
done